toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -200,7 +200,7 @@ func (im *IsoMaker) buildIsoImage() error {
 		// Directory to convert to an ISO.
 		im.buildDirPath)
 
-	return shell.ExecuteLive(false /*squashErrors*/, "mkisofs", mkisofsArgs...)
+	return shell.ExecuteLive(true /*squashErrors*/, "mkisofs", mkisofsArgs...)
```

Comment:
nit - maybe worth adding a comment here for why (same for dd)

Reply:
Fixed.